rest/answering-machine-detection/outgoing-call/outgoing-call-1.7.x.java

```diff
@@ -0,0 +1,19 @@
+// Install the Java helper library from twilio.com/docs/java/install
+import com.twilio.Twilio;
+import com.twilio.rest.api.v2010.account.Call;
+
+public class Example {
+  // Find your Account Sid and Token at twilio.com/user/account
+  public static final String ACCOUNT_SID = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+  public static final String AUTH_TOKEN = "your_auth_token";
+
+  public static void main(String[] args) {
+    Twilio.init(ACCOUNT_SID, AUTH_TOKEN);
+
+    Call call = CallUpdater("CA42ed11f93dc08b952027ffbc406d0868")
+        .setTo('+1562300000')
```

Comment:
I'm pretty sure this example won't work since strings should be enclosed in quotes. Literal characters use single quotes.